pkg/addons/adminconsole/adminconsole.go

```diff
@@ -3,20 +3,20 @@
 package adminconsole
 
 import (
-	"context"
 	"fmt"
+	"io"
+	"os"
 	"strings"
 
+	"github.com/k0sproject/dig"
+	"github.com/k0sproject/k0s/pkg/apis/v1beta1"
 	"github.com/replicatedhq/troubleshoot/pkg/apis/troubleshoot/v1beta2"
+	"github.com/sirupsen/logrus"
 	"golang.org/x/mod/semver"
-	"helm.sh/helm/v3/pkg/action"
-	"helm.sh/helm/v3/pkg/chart/loader"
-	"helm.sh/helm/v3/pkg/cli"
-	"helm.sh/helm/v3/pkg/release"
-	"sigs.k8s.io/yaml"
+	"gopkg.in/yaml.v3"
```

Comment:
This import change most likely will break the embedded license marshalling. License is a kubernetes CRD and gopkg yaml file does not couple well with those types.